docs/platforms/dotnet/guides/azure-functions-worker/index.mdx

```diff
@@ -17,71 +16,66 @@ Select which Sentry features you'd like to install in addition to Error Monitori
+// ___PRODUCT_OPTION_END___
 ```
 
 ```powershell {tabTitle:Package Manager}
-Install-Package Sentry.Azure.Functions.Worker -Version {{@inject packages.version('sentry.dotnet.azure.functions.worker') }}
+Install-Package Sentry.Extensions.Logging -Version {{@inject packages.version('sentry.dotnet.extensions.logging') }}
+// ___PRODUCT_OPTION_START___ performance
+Install-Package Sentry.OpenTelemetry -Version {{@inject packages.version('sentry.dotnet.opentelemetry') }}
+// ___PRODUCT_OPTION_END___
 ```
 
-This package extends [Sentry.Extensions.Logging](/platforms/dotnet/guides/extensions-logging/). This means that besides the Azure Functions related features, through this package you'll also get access to the `ILogger<T>` integration and also the features available in the main [Sentry](/platforms/dotnet/) SDK.
+The [Sentry.Extensions.Logging](/platforms/dotnet/guides/extensions-logging/) package also provides access to the `ILogger<T>` integration and the other core features available in the [Sentry](/platforms/dotnet/) SDK.
 
 ## Configure
 
-Sentry integration with Azure Functions is done by calling `.UseSentry()` and specifying the options, for example:
+The core features of Sentry are enabled by calling `AddSentry` when configuring logging.
 
-```csharp
-using Sentry.Azure.Functions.Worker;
-
-var builder = FunctionsApplication.CreateBuilder(args);
-
-builder.UseSentry(options =>
-{
-    options.Dsn = "___PUBLIC_DSN___";
-    // When configuring for the first time, to see what the SDK is doing:
-    options.Debug = true;
-    // Adds request URL and headers, IP and name for users, etc.
-    options.SendDefaultPii = true;
-    // ___PRODUCT_OPTION_START___ performance
-    // Set traces_sample_rate to 1.0 to capture 100% of transactions for performance monitoring.
-    // We recommend adjusting this value in production.
-    options.TracesSampleRate = 1.0;
-    // ___PRODUCT_OPTION_END___ performance
-});
-
-builder.Build().Run();
-```
-
-<Alert>
+<OnboardingOption optionId="performance">
+Performance tracing can be enabled by adding Sentry to the OpenTelemetry `TraceProviderBuilder` and then configuring Sentry itself to use OpenTelemetry.
+ </OnboardingOption>
 
-If using the ASP.NET Core integration add `UseSentry` to the `ConfigureFunctionsWebApplication` call instead.
-
-</Alert>
+For example:
 
 ```csharp
-using Sentry.Azure.Functions.Worker;
+using Sentry.OpenTelemetry;
 
 var host = new HostBuilder()
-    .ConfigureFunctionsWorkerDefaults((host, builder) =>
+    .ConfigureFunctionsWorkerDefaults()
+// ___PRODUCT_OPTION_START___ performance
+    .ConfigureServices(services =>
+    {
+        services.AddOpenTelemetry().WithTracing(builder =>
+        {
+            builder
+                .AddSentry() // <-- Configure OpenTelemetry to send traces to Sentry
+                .AddHttpClientInstrumentation(); // From OpenTelemetry.Instrumentation.Http, instruments outgoing HTTP requests
```

Comment:
question: Should we add a `PackageReference` to `OpenTelemetry.Instrumentation.Http` in the **Install** section, as mentioned by the comment?

Or is the comment good enough?